plugin/builtin/sortordertransformer/SortOrderTransformer.go

```diff
@@ -72,7 +72,7 @@ func (p *plugin) Transform(m resmap.ResMap) (err error) {
 	p.applyDefaults()
 	err = p.validate()
 	if err != nil {
-		return errors.WithStack(err)
```

Comment:
Could not get stack output using this locally. It was recently introduced and not release so removing the with stack functionality for now.